leapp/cli/upgrade/__init__.py

```diff
@@ -270,6 +268,24 @@ def preupgrade(args):
         sys.exit(1)
 
 
+@command('answer', help='Manage answerfile')
+@command_opt('answerfile', help='Path to an answerfile to update')
+@command_opt('section', action='append', metavar='dialog_sections',
+             help='Record answer for specific section in answerfile')
+def answer(args):
+    """A command to manage answerfile. Updates answerfile with userchoices"""
+    cfg = get_config()
+    if args.section:
+        args.section = list(itertools.chain(*[i.split(',') for i in args.section]))
+    answerfile_path = args.answerfile or cfg.get('report', 'answerfile')
```

Comment:
A "file exists check" won't hurt, will be done in the next commit

Reply:
And no section chosen should result in an error, not a trace. Also will be fixed